bpf/lib/policy.h

```diff
@@ -284,13 +284,13 @@ __policy_can_access(const void *map, struct __ctx_buff *ctx, __u32 local_id,
 static __always_inline int
 policy_can_access_ingress(struct __ctx_buff *ctx, __u32 src_id, __u32 dst_id,
 			  __u16 dport, __u8 proto, bool is_untracked_fragment,
-			  __u8 *match_type, __u8 *audited, __u16 *proxy_port)
+			  __u8 *match_type, __u8 *audited, __s8 *ext_err, __u16 *proxy_port)
```

Comment:
Nit: Could you please add this new argument to the comment above so it doesn't get any further out of sync.

Reply:
done. hope we can save a re-test :) (-> separate commit)